modules/inflate_geometry.py

```diff
@@ -0,0 +1,98 @@
+"""Functions for manipulating polygon geometry"""
+
+import math
+import numpy as np
+
+
+def inflate_polygon(vertices: np.ndarray, scale_distance: int) -> np.ndarray:
+    """
+    Given a list of vertices representing a polygon geometry, offset the vertices such that the perpendicular
+    distance between the original and offset edges is equal to the scale distance.
+
+    Parameters
+    -----------
+    vertices: np.ndarray
+        List of vertices
+    scale_distance: Distance (in meters) to offset the vertices by
+
+    Returns
+    -------
+    np.ndarray: List of the offset vertices
+    """
+    # Referance radius for altitude (Radius of earth = 6371 km):
+    r_ref = 6371 * (10**3)
+
+    # Convert LLA coordinates from degrees to radians:
```

Comment:
with LLA, does it work in 3d? or does it just keep the same altitude for waypoints but expand the latlon

Reply:
Yes it works in 3D, so altitude will be expanded.